assets/js/main.js

```diff
@@ -1,24 +1,23 @@
+import {cities} from "./list-of-cities.js";
+
+const searchInput = document.getElementById("search-input");
+
 const cityDropdown = document.getElementById("cities");
 const appId = "e0cb24360c821c3571a49c9d05be0fb1";
 
-const cities = [
-  { name: "Rasht", countryCode: "IR" },
-  { name: "Tehran", countryCode: "IR" },
-  { name: "Stockholm", countryCode: "SE" },
-];
-
 function generateValueFromCity(city) {
   return `${city.name},${city.countryCode}`;
 }
 
-function createCitiesOptions(cities) {
-  let optionsHtml = "";
-  cities.forEach((city) => {
-    optionsHtml += `<option value="${generateValueFromCity(city)}">${
-      city.name
-    }</option>`;
+searchInput.addEventListener("keyup", function() {
+  console.log(filterCitiesByName(searchInput.value.toLowerCase(), cities))
```

Comment:
please implement the expected behavior here instead of `console.log`

Reply:
This probably should show the filtered result to the end user.

Reply:
Move the `.toLowerCase()` part into the function.

Reply:
Done.